src/main/java/dev/jbang/cli/Alias.java

```diff
@@ -95,7 +92,7 @@ public Integer doCall() {
 				catFile = AliasUtil.addNearestAlias(null, name, scriptOrFile, description, userParams, properties);
 			}
 			info(String.format("Alias '%s' added to '%s'", name, catFile));
-		} catch (IOException e) {
+		} catch (Exception e) {
```

Comment:
why need need for catching this broad?

Reply:
For some reason I thought multi-catch wasn't part yet of Java 8 and I hate repeating the same code in multiple catch-blocks :-) will change

Reply:
In fact that code doesn't throw an exception anymore...